src/main/resources/openapi/transcriptions.yaml

```diff
@@ -681,7 +764,11 @@ components:
         description:
           type: string
           example: Standard
-
+    UpdateTranscriptions400ErrorCode:
+      type: string
```

Comment:
Not sure about mixing underscore and hyphen in the error type e.g. `TRANSCRIPTION_UT-PP-400`.
It looks strange to me

Reply:
wasn't sure what best to do here, but have changed it to code 112